rtv/utils.py

```diff
@@ -118,17 +139,29 @@ def validate(ch):
 def display_message(stdscr, message):
```

Comment:
You can just make the standard behavior to accept a list, instead of trying to switch between use cases. This is only being used in one other place, which would be easy enough to change to
```python
display_message(self.stdscr, ['Invalid Subreddit'])